moveit_core/collision_detection_fcl/src/collision_common.cpp

```diff
@@ -96,16 +98,15 @@ bool collisionCallback(fcl::CollisionObjectd* o1, fcl::CollisionObjectd* o2, voi
       {
         always_allow_collision = true;
         if (cdata->req_->verbose)
-          ROS_DEBUG_NAMED(
-              "collision_detection.fcl", "Collision between '%s' (type '%s') and '%s' (type '%s') is always allowed. "
+          RCLCPP_DEBUG(logger, "Collision between '%s' (type '%s') and '%s' (type '%s') is always allowed. "
```

Comment:
```suggestion
          RCLCPP_DEBUG(LOGGER, "Collision between '%s' (type '%s') and '%s' (type '%s') is always allowed. "
```

Reply:
We have changed the logger name, this should be fixed https://github.com/ros-planning/moveit2/commit/a3011f83f7a76ca3856b5e8b490f587b1af43b7d